CONTRIBUTING.md

```diff
@@ -91,7 +93,13 @@ Add new code to Julia's base libraries as follows:
 
  4. Include your tests in `test/Makefile` and `test/runtests.jl`.
 
-Build as usual, and do `make clean testall` to test your contribution. If your contribution includes changes to Makefiles or external dependencies, make sure you can build Julia from a clean tree using `git clean -fdx` or equivalent (be careful – this command will delete any files lying around that aren't checked into git). Make sure that [Travis](http://www.travis-ci.org) greenlights the pull request with a `Good to merge` message.
+Build as usual, and do `make clean testall` to test your contribution. If your contribution includes changes to Makefiles or external dependencies, make sure you can build Julia from a clean tree using `git clean -fdx` or equivalent (be careful – this command will delete any files lying around that aren't checked into git).
+
+Note: You can run specific test files, e.g. to run `test/bitarray.jl` and `test/math.jl`:
+
+    julia test/runtests.jl bitarray math
```

Comment:
Have amended to mention both, haven't gone into detail e.g. about subfolders as I don't know the details!

*Used `./julia` though of course this probably won't be correct on Windows...* Thanks!